src/cloud/components/DocPage/NewDocContextMenu.tsx

```diff
@@ -210,7 +208,7 @@ const DocContextMenu = ({
                     >
                       {contributorsState.sliced > 0
                         ? `+${contributorsState.sliced}`
-                        : '-'}
+                        : ''}
```

Comment:
it makes users impossible to minimize once they showed all contributors. Let's revert it.

Reply:
Reverted